src/SessionSetup/ConnectToStadiaWidget.cpp

```diff
@@ -350,38 +268,16 @@ void ConnectToStadiaWidget::SetupStateMachine() {
   s_connected_.addTransition(this, &ConnectToStadiaWidget::ErrorOccurred, &s_instance_selected_);
 }
 
-void ConnectToStadiaWidget::ReloadInstances() {
-  CHECK(ggp_client_ != nullptr);
-  instance_model_.SetInstances({});
-
-  orbit_ggp::Client::InstanceListScope scope =
-      ui_->allInstancesCheckBox->isChecked()
-          ? orbit_ggp::Client::InstanceListScope::kAllReservedInstances
-          : orbit_ggp::Client::InstanceListScope::kOnlyOwnInstances;
-
-  ggp_client_->GetInstancesAsync(scope, selected_project_)
-      .Then(main_thread_executor_.get(), [this](ErrorMessageOr<QVector<Instance>> instances) {
-        OnInstancesLoaded(std::move(instances));
-      });
-
-  ggp_client_->GetProjectsAsync().Then(
-      main_thread_executor_.get(),
-      [this](ErrorMessageOr<QVector<Project>> projects) { OnProjectsLoaded(std::move(projects)); });
-}
-
 void ConnectToStadiaWidget::LoadCredentials() {
   CHECK(selected_instance_.has_value());
 
-  const std::string instance_id = selected_instance_->id.toStdString();
+  const std::string& instance_id{selected_instance_->id.toStdString()};
```

Comment:
This should probably not be a reference?

Reply:
done